airflow/models/dag.py

```diff
@@ -1197,7 +1197,7 @@ def param(self, name: str, default: Any = NOTSET) -> DagParam:
 
         :param name: dag parameter name.
         :param default: fallback value for dag parameter.
-        :return: DagParam instance for specified name and current dag.
+        :return: Dag Param instance for specified name and current dag.
```

Comment:
```suggestion
        :return: DagParam instance for specified name and current dag.
```
I think it should be DagParam?

Reply:
I added space it as I did not want to add it to spelling.txt :)

Reply:
Fixed properly

Reply:
Sorry. You're correct! it's now Param not DagParam.